src/app/utils/testing.utils.ts

```diff
@@ -0,0 +1,25 @@
+import { FormGroupDirective, FormGroup } from '@angular/forms';
```

Comment:
copy the entire file from ADR so you can start using similar approach for your mockiing